src/App.tsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
-
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './components/PostList';
+
+const preparedPosts: PreparedPost[] = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId) || null,
+  comments: comments.filter(comment => comment.postId === post.id) || null,
```

Comment:
could `null` ever be set to comments? if there are no comments, it will be just an empty array